src/kOS/Binding/FlightControlManager.cs

```diff
@@ -201,6 +201,8 @@ public void SelectAutopilotMode(object autopilotMode)
             if (autopilotMode is Direction)
             {
                 //TODO: implment use of direction subclasses.
+                throw new Safe.Exceptions.KOSException(
```

Comment:
It may be possible to use the old-school stability assist (that doesn't reset itself when too far from the target rotation), but it wouldn't really help in case of typo, and there's cooked steering for just using a direction anyway.